javascript/ql/test/query-tests/Security/CWE-078/lib/lib.js

```diff
@@ -324,4 +324,14 @@ module.exports.typeofcheck = function (name) {
 module.exports.typeofcheck = function (arg) {
 	var cmd = "MyWindowCommand | findstr /i /c:" + arg; // NOT OK
 	cp.exec(cmd); 
-}
\ No newline at end of file
+}
+
+function id(x) {
+	return x;
+}
+
+module.exports.id = id;
+
+module.exports.unproblematic = function() {
+	cp.exec("rm -rf " + id("test")); // OK [INCONSISTENCY]
```

Comment:
I had to annotate this as `[INCONSISTENCY]` since the result is only filtered away at the level of the query, not the configuration.